pkg/controller/storagecluster/generate.go

```diff
@@ -41,3 +41,13 @@ func generateNameForCephFilesystemSC(initData *ocsv1.StorageCluster) string {
 func generateNameForCephBlockPoolSC(initData *ocsv1.StorageCluster) string {
 	return fmt.Sprintf("%s-ceph-rbd", initData.Name)
 }
+
+// generateNameForSnapshotClass function generate 'SnapshotClass' name.
+// 'snapshotType' can be: 'rbd' or 'cephfs'
+func generateNameForSnapshotClass(initData *ocsv1.StorageCluster, snapshotType string) string {
+	return fmt.Sprintf("%s-%splugin-snapclass", initData.Name, snapshotType)
+}
+
+func generateNameForSnapshotClassDriver(initData *ocsv1.StorageCluster, snapshotType string) string {
```

Comment:
why we need this one? don't we already have a driver for storage class?

Reply:
yeah @aruniiird looks like a stale func now. please get rid of `generateNameForSnapshotClassDriver`